webapp/javascript/components/TimelineChart/TimelineChartSelection.ts

```diff
@@ -20,6 +20,7 @@ type PlotType = {
   clearSelection: (preventEvent: boolean) => void;
   setSelection: (ranges: any, preventEvent: any) => void;
   getSelection: () => {} | null;
+  setCursor: (cursor: string) => void;
```

Comment:
```suggestion
  setCursor: (cursor: 'grab' | 'crosshair') => void;
```

If possible let's narrow the types as much as possible. By having just `string` thereotically a `setCursor("FOOOOOOO")` is valid, although it's obviously not true.